src/site/content/en/blog/declarative-link-capturing/index.md

```diff
@@ -152,6 +154,51 @@ possible for the site to automatically trigger it. Second, the capability to foc
 window on its own domain and fire an event containing the clicked URL. This is intended to allow the
 site to navigate an existing window to a new page, overriding the default HTML navigation flow.
 
+## Migrate to Launch Handler API {: #migration }
+
+The Declarative Link Capturing API [origin trial](https://developer.chrome.com/origintrials/#/view_trial/4285175045443026945)
+is set to [expire on March 30, 2022](https://groups.google.com/a/chromium.org/g/blink-dev/c/2c4bul4V3GQ/m/Anluh1txBQAJ)
+for Chrome M97 and below. It will be replaced by a set of [new features and APIs](https://docs.google.com/document/d/1w9qHqVJmZfO07kbiRMd9lDQMW15DeK5o-p-rZyL7twk/edit)
```

Comment:
```suggestion
for Chromium&nbsp;97 and below. It will be replaced by a set of [new features and APIs](https://docs.google.com/document/d/1w9qHqVJmZfO07kbiRMd9lDQMW15DeK5o-p-rZyL7twk/edit)
```